app/src/main/res/layout/component_tabstray.xml

```diff
@@ -24,6 +24,15 @@
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintWidth_percent="0.1" />
 
+
+    <androidx.constraintlayout.widget.ConstraintLayout
+        android:id="@+id/infoBanner"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        android:background="@color/foundation_normal_theme"
+        app:layout_constraintTop_toBottomOf="@+id/topBar">
+    </androidx.constraintlayout.widget.ConstraintLayout>
```

Comment:
nit - I don't think we need open/closing tag here since there's nothing in the middle?